lib/fog/ovirt/requests/compute/v4/update_vm.rb

```diff
@@ -0,0 +1,45 @@
+module Fog
+  module Compute
+    class Ovirt
+      class V4
+        class Real
+          # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+          def update_vm(attrs)
+            attrs = attrs.dup
+            vm_service = client.system_service.vms_service.vm_service(attrs[:id])
+
+            if attrs[:cores].present?
+              cpu_topology = OvirtSDK4::CpuTopology.new(:cores => attrs[:cores], :sockets => "1")
+              attrs[:cpu] = OvirtSDK4::Cpu.new(:topology => cpu_topology)
+            end
+            attrs[:cluster] = attrs[:cluster].present? ? client.system_service.clusters_service.cluster_service(attrs[:cluster]).get : nil
+            attrs[:template] = if attrs[:template].present?
+                                 client.system_service.templates_service.template_service(attrs[:template]).get
+                               else
+                                 client.system_service.get.special_objects.blank_template
+                               end
```

Comment:
Lines 11-20 appear in `create_vm` as well. The only joint place I found so far for them is the Shared module in `compute/v4.rb` andI don't think that is the right place for it.
I could add some request to include in both but maybe there is a better way.